custom/icds_reports/reports/service_delivery_dashboard.py

```diff
@@ -65,6 +65,7 @@ def base_data(row_data):
             )
         else:
             return dict(
+                num_launched_awcs=get_value_or_data_not_entered(row_data, 'num_launched_awcs'),
```

Comment:
does this mean the data was just missing before?

Reply:
Yes It was not missing because this check was not being applied before. its a new check request. which it to check whether the location has any launched awc or not